java-checks/src/main/java/org/sonar/java/checks/EnumMutableFieldCheck.java

```diff
@@ -61,9 +63,11 @@ public void visitNode(Tree tree) {
     ClassTree enumTree = (ClassTree) tree;
     for (Tree member : enumTree.members()) {
       if (member.is(Tree.Kind.VARIABLE)) {
-        VariableTree variableTree = (VariableTree) member;
-        ModifierKeywordTree publicModifier = ModifiersUtils.getModifier(variableTree.modifiers(), Modifier.PUBLIC);
-        if (publicModifier != null && !ModifiersUtils.hasModifier(variableTree.modifiers(), Modifier.STATIC)) {
+        ModifiersTree modifiers = ((VariableTree) member).modifiers();
+        ModifierKeywordTree publicModifier = ModifiersUtils.getModifier(modifiers, Modifier.PUBLIC);
+        if (publicModifier != null && !ModifiersUtils.hasModifier(modifiers, Modifier.STATIC)
+        // FIXME SONARJAVA-1604 final mutable field should raise issues
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1134)